[2024-02-02 06:20:20,426] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_26.aggregation_test scheduled__2024-02-01T06:19:57.633868+00:00 [queued]>
[2024-02-02 06:20:20,444] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: project_6_26.aggregation_test scheduled__2024-02-01T06:19:57.633868+00:00 [queued]>
[2024-02-02 06:20:20,445] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 06:20:20,445] {taskinstance.py:1357} INFO - Starting attempt 1 of 2
[2024-02-02 06:20:20,445] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2024-02-02 06:20:20,490] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): aggregation_test> on 2024-02-01 06:19:57.633868+00:00
[2024-02-02 06:20:20,504] {standard_task_runner.py:52} INFO - Started process 4682 to run task
[2024-02-02 06:20:20,509] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'project_6_26', 'aggregation_test', 'scheduled__2024-02-01T06:19:57.633868+00:00', '--job-id', '410', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmp_mwe77s5', '--error-file', '/tmp/tmpwyzgkpz_']
[2024-02-02 06:20:20,511] {standard_task_runner.py:80} INFO - Job 410: Subtask aggregation_test
[2024-02-02 06:20:20,768] {task_command.py:369} INFO - Running <TaskInstance: project_6_26.aggregation_test scheduled__2024-02-01T06:19:57.633868+00:00 [running]> on host 20775c919683
[2024-02-02 06:20:21,075] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=project_6_26
AIRFLOW_CTX_TASK_ID=aggregation_test
AIRFLOW_CTX_EXECUTION_DATE=2024-02-01T06:19:57.633868+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-02-01T06:19:57.633868+00:00
[2024-02-02 06:20:21,269] {logging_mixin.py:115} INFO -    id  kode_prov  case_id     tanggal  total
0   1         32        1  2020-08-05   1815
[2024-02-02 06:20:21,294] {logging_mixin.py:115} INFO -    kode_prov  id  case_id  year  total
0         32  10        1  2020   2507
[2024-02-02 06:20:21,295] {python.py:173} INFO - Done. Returned value was: None
[2024-02-02 06:20:21,370] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=project_6_26, task_id=aggregation_test, execution_date=20240201T061957, start_date=20240202T062020, end_date=20240202T062021
[2024-02-02 06:20:21,419] {local_task_job.py:156} INFO - Task exited with return code 0
[2024-02-02 06:20:21,562] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
